core/domain/collection_services.py

```diff
@@ -540,6 +540,39 @@ def get_collection_summaries_matching_ids(collection_ids):
             collection_ids)]
 
 
+def get_nodes_in_playable_order(nodes_list):
+    """Returns the nodes list in playable order.
+
+    Args:
+        nodes_list: list(dict). An unordered list of collection nodes.
+
+    Raises:
+        Exception: Unable to sort the collection nodes in playable order.
+
+    Returns:
+        list(dict). A list of collection nodes in playable order.
+    """
+    unsorted_nodes = nodes_list
+    accumulated_skill_ids = set()
+    sorted_nodes = []
+    while len(unsorted_nodes):
+        for index, node in enumerate(unsorted_nodes):
+            prerequisite_skill_ids = set(node["prerequisite_skill_ids"])
```

Comment:
Use single quotes in python; ditto elsewhere.

Reply:
Done! :)
(Thanks!)